samples/snippets/seek_lite_subscription_example.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This application demonstrates how to initiate an out-of-band seek for a
+subscription with the Pub/Sub Lite API.
+"""
+
+import argparse
+
+
+def seek_lite_subscription(project_number, cloud_region, zone_id, subscription_id, seek_target, wait_for_operation):
+    # [START pubsublite_seek_subscription]
+    from google.api_core.exceptions import NotFound
+    from google.cloud.pubsublite import AdminClient
+    from google.cloud.pubsublite.types import CloudRegion, CloudZone, SubscriptionPath
+
+    # TODO(developer):
+    # project_number = 1122334455
+    # cloud_region = "us-central1"
+    # zone_id = "a"
+    # subscription_id = "your-subscription-id"
+    # seek_target = BacklogLocation.BEGINNING
+    # wait_for_operation = True
+
+    cloud_region = CloudRegion(cloud_region)
+    location = CloudZone(cloud_region, zone_id)
+    subscription_path = SubscriptionPath(project_number, location, subscription_id)
+
+    client = AdminClient(cloud_region)
+    try:
+        seek_operation = client.seek_subscription(subscription_path, seek_target)
+        print(f"Seek operation: {seek_operation.operation.name}")
+    except NotFound:
+        print(f"{subscription_path} not found.")
+        return
+
+    # Optional: Wait for the seek operation to complete, which indicates when
+    # subscribers for all partitions are receiving messages from the seek
+    # target.
```

Comment:
Would you mind move some of the comment to where users set `seek_target`?

Reply:
I added the comments just below the `TODO(developer)` block.